tests/testPhotoCal.py

```diff
@@ -87,38 +120,77 @@ def getAstrometrySolution(self, loglvl = Log.INFO):
         )
 
     def testGetSolution(self):
-        res = self.getAstrometrySolution(loglvl=Log.DEBUG)
-        self.assertTrue(res is not None)
-        self.assertGreater(len(res.matches), 50)
+        self.assertTrue(self.res is not None)
+        self.assertGreater(len(self.res.matches), 50)
 
     def test1(self):
-        res = self.getAstrometrySolution()
-        matches = res.matches
 
         print 'Test1'
 
-        logLevel = Log.DEBUG
-        log = Log(Log.getDefaultLog(),
-                  'testPhotoCal',
-                  logLevel)
+        task = PhotoCalTask(config=self.config, schema=self.schema)
+        pCal = task.run(exposure=self.exposure, matches=self.matches)
+        print "Ref flux fields list =", pCal.arrays.refFluxFieldList
```

Comment:
Don't know if this is common. I would expect it should either be logged as info or silenced. But again I don't know what is common practice in tests.